packages/perseus/src/widgets/radio/__stories__/choice-icon.stories.jsx

```diff
@@ -1,161 +1,242 @@
+export const Correct = (args: StoryArgs): React.Node => {
     return (
-        <ChoiceIcon
-            {...defaultObject}
-            pos={0}
-            showCorrectness={true}
-            checked={true}
-            reviewMode={false}
-        />
+        <>
+            <p>Library</p>
+            <ChoiceIcon
+                {...defaultProps}
+                checked={true}
+                correct={true}
+                showCorrectness={true}
+                reviewMode={true}
+            />
+            <ChoiceIcon
+                {...defaultProps}
+                checked={true}
+                correct={true}
+                showCorrectness={true}
+                reviewMode={true}
+                multipleSelect={true}
+            />
+            <p>SAT</p>
+            <ChoiceIcon
+                {...defaultSATProps}
+                checked={true}
+                correct={true}
+                showCorrectness={true}
+                reviewMode={true}
+            />
+            <ChoiceIcon
+                {...defaultSATProps}
+                checked={true}
+                correct={true}
+                showCorrectness={true}
+                reviewMode={true}
+                multipleSelect={true}
+            />
+        </>
     );
 };
 
-export const ShowCorrectnessIncorrect = (args: StoryArgs): React.Node => {
+export const Incorrect = (args: StoryArgs): React.Node => {
     return (
-        <ChoiceIcon
-            {...defaultObject}
-            pos={0}
-            correct={false}
-            showCorrectness={true}
-            reviewMode={false}
-        />
+        <>
+            <p>Library</p>
+            <ChoiceIcon
+                {...defaultProps}
+                checked={true}
+                correct={false}
+                showCorrectness={true}
+                reviewMode={true}
+            />
+            <ChoiceIcon
+                {...defaultProps}
+                checked={true}
+                correct={false}
+                showCorrectness={true}
+                reviewMode={true}
+                multipleSelect={true}
+            />
+            <p>SAT</p>
+            <ChoiceIcon
+                {...defaultSATProps}
+                checked={true}
+                correct={false}
+                showCorrectness={true}
+                reviewMode={true}
+            />
+            <ChoiceIcon
+                {...defaultSATProps}
+                checked={true}
+                correct={false}
+                showCorrectness={true}
+                reviewMode={true}
+                multipleSelect={true}
+            />
+        </>
     );
 };
 
-export const CrossedOut = (args: StoryArgs): React.Node => {
-    return <ChoiceIcon {...defaultObject} pos={0} crossedOut={true} />;
-};
-
-export const CrossedOutPressed = (args: StoryArgs): React.Node => {
+export const AllPositions = (args: StoryArgs): React.Node => {
+    const arr26 = Array(26).fill();
```

Comment:
**suggestion:** `AllLetters` might make more sense to people.

Reply:
At first I was like "yeah totally!" but now I'm wondering if it would be more confusing because in reality it's an array of `undefined` 26 times.

Also as I'm typing this I'm getting the most intense deja vu... 🤔

Reply:
The choices that are rendered are `A`, `B`, `C`, etc. right?

Reply:
Yeah, but the array itself doesn't contain anything but undefined. It was just shorthand for `for(let i = 0; i < 26; i++)`. Doesn't matter, I don't feel particularly strongly about this so I went ahead and changed it.